src/GT.Data/Data/GTAppDb/Entities/ListingInquiry.cs

```diff
@@ -9,10 +8,10 @@ public class ListingInquiry : IGTEntity
 		public string Id { get; set; }
 
 		[Column(TypeName = "nvarchar(100)")]
-		public string MessageTitle { get; set; }
+		public string? MessageTitle { get; set; }
 
 		[Column(TypeName = "nvarchar(500)")]
-		public string MessageBody { get; set; }
+		public string? MessageBody { get; set; }
 
 		[Column(TypeName = "nvarchar(100)")]
```

Comment:
Propose nvarchar(254).

Reply:
After an internal discussion it was decided that 500 was to be kept until further notice. Resolving comment.

Reply:
Please re-read.